### PR TITLE
Add Tempesta Technologies

### PR DESCRIPTION
**Pull request by aadeshmodak13**
added new company - Tempesta Technologies #1685

### CODE REVIEW

**Comment by KieranRobson** [comment]
Please add it to the README aswell.

**Comment by dougaitken** [comment]
@aadeshmodak13  When you create a Pull Request in this this repository there is a Pull Request template with a checklist - you've deleted this when submitting this PR.

If you had left that checklist and reviewed each option you would have seen that there is mention of what @KieranRobson has asked for. 

https://github.com/remoteintech/remote-jobs/blob/7f10ae795f64c19b24c2bebba7c51629a0103b86/.github/PULL_REQUEST_TEMPLATE.MD?plain=1#L29

I've added some comments to the profile for you to fix. Thanks.

File: company-profiles/tempesta.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Tempesta Technologies Inc.
+
+## Company blurb
+   For fast growing and innovative CDNs that want to deliver the best performance and security, Tempesta Technologies provides customizable solutions using the open source for secure low-latency and high throughput content delivery. This allows Internet companies to deliver the best user experience and not depend on vendor’s proprietary products or third party services.
+
+## Comapny size
```
Comment (dougaitken) [inline company-profiles/tempesta.md:6]: Spelling - `Company size`

File: company-profiles/tempesta.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+11-50 employees
+
+## remote status
```
Comment (dougaitken) [inline company-profiles/tempesta.md:9]: Capitalise the first letter. `Remote status`

File: company-profiles/tempesta.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+   work remotely with a flexible schedule using Scrum and Kanban ideas and specialize in development of low level high-performance networking and data storage systems.Since 2008 and during these years many international high tech companies became our clients, e.g. MariaDB, Percona, SanDisk, Skype,
+   
+
+## region
```
Comment (dougaitken) [inline company-profiles/tempesta.md:13]: Capitalise - `Region`

File: company-profiles/tempesta.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+   Seattle,WA
+
+## company technologies
```
Comment (dougaitken) [inline company-profiles/tempesta.md:16]: Capitalise the first letter. `Remote status`

File: company-profiles/tempesta.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    Linux kernel, Databases, Web Application Firewalls, HTTP load balancers, DDoS mitigation, High performance applications, Layer 7 DDoS, Bot Blocking, Web application security, Website acceleration, Web content protection, and C++
+
+
+## office locations
```
Comment (dougaitken) [inline company-profiles/tempesta.md:20]: Capitalise the first letter. `Office locations`

File: company-profiles/tempesta.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+## company technologies
+    Linux kernel, Databases, Web Application Firewalls, HTTP load balancers, DDoS mitigation, High performance applications, Layer 7 DDoS, Bot Blocking, Web application security, Website acceleration, Web content protection, and C++
+
+
+## office locations 
+   1) Seattlw ,WA
+   2) Yerevan ,AM
+   3) London, England
+
+## how to apply
```
Comment (dougaitken) [inline company-profiles/tempesta.md:25]: Capitalise the first letter. `How to apply`

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
Closing, no author reply.

**Pull request closed without merging**